app/src/main/java/com/cornellappdev/transit/ui/viewmodels/RouteViewModel.kt

```diff
@@ -62,6 +63,11 @@ class RouteViewModel @Inject constructor(
 
     val lastRouteFlow: StateFlow<ApiResponse<RouteOptions>> = routeRepository.lastRouteFlow
 
+    /**
+     * Default map location
+     */
+    val defaultIthaca = LatLng(42.44, -76.50)
```

Comment:
Hmmm add a TODO to make this set with an `init` block so that it initializes on the route... we may have to make this something else other than just a `val` to get that to work.

Reply:
Will do!